### PR TITLE
3.2.0 feature ragbaby

### PR DESCRIPTION
**Pull request by olliklee**
completely refactored and separated

### CODE REVIEW

File: lib/application/i18n/assets/en.json [hunk 1]
```diff
@@ -4004,6 +4004,12 @@
   "rabbit_description": "(Secret) Rabbit is a high-speed stream cipher",
   "rabbit_initialization_vector": "Initialization Vector (Optional)",
   "rabbit_initialization_vector_format": "Initialization vector type does not match the initialization vector!",
+  "ragbaby_title": "Ragbaby Cipher",
+  "ragbaby_description": "Ragbaby is poly-alphabetic substitution cipher with a 24 letters alphabet. J=>I and X=>U. The rotation changes for each word and letter.",
```
Comment (S-Man42) [inline lib/application/i18n/assets/en.json:4008]: I would shorten it to "Poly-alphabetic substitution cipher with a 24 letters alphabet"

Comment (olliklee) [reply lib/application/i18n/assets/en.json]: Done

**Comment by S-Man42** [comment]
@olliklee Pls add sources to registry :)

**Comment by olliklee** [comment]
done

**Pull request merged**